packages/block-editor/src/components/index.js

```diff
@@ -54,7 +54,7 @@ export {
 export { default as __experimentalImageSizeControl } from './image-size-control';
 export {
 	default as InnerBlocks,
-	useInnerBlocksProps as __experimentalUseInnerBlocksProps,
+	useInnerBlocksProps as useInnerBlocksProps,
```

Comment:
NP: no need to alias now. :)

Reply:
Oops. We should have the linter catch stuff like this.

Reply:
Good idea. I was surprised it didn't, actually, but I guess it isn't common :shrug: